Table component issue #80

AdhamAH opened the pull request:
I had some trouble with the old branch so I made a new on

@SomeMoosery I hope I did it right this time 
![1](https://user-images.githubusercontent.com/55054963/77212245-b04c7d00-6b06-11ea-8311-f1e32b7add76.gif)

AdhamAH commented:
Actually, in this one, I forget to make the changes in the gif! I am sick and sleepy, sorry! that's why it passed I guess

tesla809 commented:
You are doing great @AdhamAH ! Amazing stuff! Feel better and get some rest. We can look over this next time. Brillant code and great suggestion to use material-ui!

AdhamAH commented:
I could only sleep for 3.5 hours  😢 so I worked on this a little

I am having problems dealing with git and how to pull and push so I hope I did it right.

First I changed the style in DiagnosticContainer and also changed US-EN zone to be undefined, this will take the local system format and apply it I think, please can try and let me know
![style](https://user-images.githubusercontent.com/55054963/77221682-232c1700-6b4c-11ea-9707-00e0deedeb3a.gif)

second I removed the date and tabs from table.js and will try to make a toggle component soon 

![table](https://user-images.githubusercontent.com/55054963/77221744-d5fc7500-6b4c-11ea-9fbd-cb5c7b27c4de.gif)

AdhamAH commented:
![Screenshot 2020-03-21 at 08 50 33](https://user-images.githubusercontent.com/55054963/77222177-3f7e8280-6b51-11ea-88c9-981cb676f888.png)

SomeMoosery commented on client/src/css/Table.css:
```diff
@@ -0,0 +1,78 @@
+        width: 70px;
+    }
+}
+<<<<<<< HEAD
```
It looks like there's a merge conflict tag in here - we just need to rebase off of master, I can add some instructions in my overall comment on the typical strategy to do that!

AdhamAH replied:
I did push, not sure if that what you wanted, let me know

AdhamAH commented on client/src/components/Table.js:
```diff
@@ -1,12 +1,9 @@
 import React from "react";
```
@tesla809 I just remimbered that you asked e to use
`import React, {Component} from 'react`
I can do that but may I ask why? what is the difference?
